setup.py

```diff
@@ -10,7 +10,7 @@
 general_requirements = [
     'dpath >= 1.5.0, < 2.0.0',
     'pytest >= 4.4.1, < 6.0.0',  # For openfisca test
-    'numpy >= 1.11, < 1.18',
+    'numpy >= 1.18, < 1.19',
```

Comment:
It shouldn't be necessary to bump the minimum version, all your changes above will keep working in old numpy versions.

Reply:
Yes, thx!
> + the changes affect the `select` function that [changed before numpy 1.11](https://github.com/numpy/numpy/pull/14583)